ts/features/idpay/payment/screens/IDPayPaymentAuthorizationScreen.tsx

```diff
@@ -59,39 +69,207 @@ const IDPayPaymentAuthorizationScreen = () => {
       contextualHelp={emptyContextualHelp}
+      goBack={false}
     >
-      <LoadingSpinnerOverlay isLoading={isLoading} loadingOpacity={100}>
-        <SafeAreaView style={IOStyles.flex}>
-          <View style={IOStyles.flex}>
-            <ContentWrapper>{content}</ContentWrapper>
-          </View>
-          <FooterWithButtons
-            type="TwoButtonsInlineHalf"
-            leftButton={{
-              title: "Annulla",
-              bordered: true,
-              onPress: handleCancel,
-              disabled: isUpserting
-            }}
-            rightButton={{
-              title: isUpserting ? "" : "Conferma",
-              onPress: handleConfirm,
-              isLoading: isUpserting,
-              disabled: isUpserting
-            }}
-          />
-        </SafeAreaView>
-      </LoadingSpinnerOverlay>
+      <SafeAreaView style={IOStyles.flex}>
+        <View style={IOStyles.flex}>
+          <ContentWrapper>
+            <H1>{I18n.t("idpay.payment.authorization.header")}</H1>
+            <VSpacer size={24} />
+            {renderContent()}
+          </ContentWrapper>
+        </View>
+        <FooterWithButtons
+          type="TwoButtonsInlineHalf"
+          leftButton={{
+            title: I18n.t("global.buttons.cancel"),
+            bordered: true,
+            onPress: handleCancel,
+            disabled: isUpserting
+          }}
+          rightButton={{
+            title: isUpserting ? "" : I18n.t("global.buttons.confirm"),
+            onPress: handleConfirm,
+            isLoading: isUpserting || isLoading,
+            disabled: isUpserting || isLoading
+          }}
+        />
+      </SafeAreaView>
     </BaseScreenComponent>
   );
 };
 
-export type { IDPayPaymentAuthorizationScreenRouteParams };
+const ExistingTransactionDataContent = ({
+  data
+}: {
+  data: NonNullable<AuthPaymentResponseDTO>;
+}) => {
+  const getValueOrDash = (value: string | undefined) =>
+    pipe(
+      value,
+      O.fromNullable,
+      O.getOrElse(() => "-")
+    );
+  const amountCents = formatNumberCurrency(data.amountCents / 100);
+  const reward = pipe(
+    data.reward,
+    O.fromNullable,
+    O.fold(
+      () => "-",
+      data => formatNumberCurrency(data / 100)
+    )
+  );
+  const businessName = getValueOrDash(data.businessName);
+  const initiativeName = getValueOrDash(data.initiativeName);
+  const date = pipe(
+    data.trxDate,
+    O.fromNullable,
+    O.fold(
+      () => "-",
+      date => format(new Date(date), "D MMMM YYYY, HH:mm")
+    )
+  );
+  return (
+    <>
+      <Divider />
+      <VSpacer size={16} />
+      <View style={[IOStyles.rowSpaceBetween, IOStyles.alignCenter]}>
+        <NewH6>{I18n.t("idpay.payment.authorization.toAuth")}</NewH6>
+        <H1>{reward}</H1>
+      </View>
+
+      <VSpacer size={16} />
+      <Divider />
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.amount")}
+        value={amountCents}
+        accessibilityLabel=" "
```

Comment:
```suggestion
        accessibilityLabel={I18n.t("idpay.payment.authorization.amount")}
```
Accessibility labels are missing everywhere. Please add them.